src/Organizer/Events/GeoCoordinatesUpdated.php

```diff
@@ -0,0 +1,10 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Organizer\Events;
+
+use CultuurNet\UDB3\Offer\Events\AbstractGeoCoordinatesUpdated;
+
+class GeoCoordinatesUpdated extends AbstractGeoCoordinatesUpdated
```

Comment:
This event should not extend the abstract one in the `Offer` namespace, because `Offers` = `Events` + `Places`. `Organizers` is something separate. So it's best to extend the abstract `OrganizerEvent` instead: https://github.com/cultuurnet/udb3-php/blob/master/src/Organizer/Events/OrganizerEvent.php